structured_data/2021_02_26_scikit_learn_pipelines/scikit_learn_pipelines_and_lightbm_titanic_dataset.ipynb

```diff
@@ -0,0 +1,1157 @@
+{
```

Comment:
notebook name contains typo: `lightbm` instead of `lightgbm`

Reply:
Good spot!